lib/web/css/source/lib/_responsive.less

```diff
@@ -15,7 +15,14 @@
 @media-target: 'all'; // Sets target device for styles output (all|desktop|mobile)
 
 //
-//  Media width mixin used to group styles output based on media queries
+//  Introduce Media-common mixin used to group extends global styles output.
+//  ---------------------------------------------
+
+.media-common() {
+}
```

Comment:
Could you explain a bit why it's needed?

Reply:
This allow wrap common styles (selectors, mixins) for reuseable while developement
You can see file as example of https://github.com/magento/magento2/blob/2.4-develop/app/design/frontend/Magento/blank/web/css/source/_extends.less. This file define some selector allow frontend can reuse it from anywhere in theme by :extend

Reply:
https://github.com/magento/magento2/blob/2.4-develop/app/design/frontend/Magento/luma/Magento_Customer/web/css/source/_module.less#L159 Example usage :extend

Reply:
@ihor-sviziev  Should i seperate to another PR ? only keep update less js in this and move other change to seperate PR

Reply:
Yeah! Thx for separating it!

Reply:
Updated in another PR